### PR TITLE
feature/close-on-transaction-complete-option

### LINKED ISSUE #253: Feature Request: checkbox to close or keep open Transaction Screen after clicking save
Can the editing interface be automatically closed after the transaction is submitted? thanks
image

### PR DESCRIPTION
**Pull request by hamza-fadil**
**Summary**
Close transaction after submit
-   

I added a switch to close after a transaction is complete. Sometime I missclick on submit twice and it duplicate the value, this should also close #253 
Let me know if you want to change anything 
Thank you

### CODE REVIEW

File: src/i18n/locale/translations/en-US.ts [hunk 1]
```diff
@@ -149,4 +149,6 @@ export default {
   bills_paid: 'Paid',
   bills_not_expected: 'Not expected',
   transaction_form_bill_label: 'Bill',
+  configuration_other: 'Other',
+  close_after_transation: 'Close after transaction is submitted'
```
Comment (victorbalssa) [inline src/i18n/locale/translations/en-US.ts]:
```suggestion
  close_after_transaction: 'Close transaction form after submission'
```

File: src/i18n/locale/translations/en-US.ts [hunk 2]
```diff
@@ -149,4 +149,6 @@ export default {
   bills_paid: 'Paid',
   bills_not_expected: 'Not expected',
   transaction_form_bill_label: 'Bill',
+  configuration_other: 'Other',
```
Comment (victorbalssa) [inline src/i18n/locale/translations/en-US.ts]:
```suggestion
  configuration_transaction_form: 'Transaction Form',
```

File: src/i18n/locale/translations/fr-FR.ts [hunk 3]
```diff
@@ -147,5 +147,6 @@ export default {
   bills_paid: 'Payé',
   bills_not_expected: 'Non attendu',
   transaction_form_bill_label: 'Facture',
-  
+  configuration_other: 'Autre',
```
Comment (victorbalssa) [inline src/i18n/locale/translations/fr-FR.ts]:
```suggestion
  configuration_transaction_form: 'Formulaire',
```

File: src/i18n/locale/translations/fr-FR.ts [hunk 4]
```diff
@@ -147,5 +147,6 @@ export default {
...
+  close_after_transaction: 'Fermer la transaction après l\'avoir envoyer'
```
Comment (victorbalssa) [inline src/i18n/locale/translations/fr-FR.ts]:
```suggestion
  close_after_transaction: 'Fermer le formulaire après l\'avoir envoyé'
```

File: src/components/Screens/ConfigurationScreen.tsx [hunk 5]
```diff
@@ -273,6 +280,31 @@ export default function ConfigurationScreen({ navigation }: ScreenType) {
           <Ionicons name={Platform.select({ ios: 'logo-apple-appstore', android: 'logo-google-playstore' })} size={23} color="gray" />
         </APressable>
       </AView>
+      <AText py={10} px={10} fontSize={18} bold>
+        {translate('configuration_other')}
```
Comment (victorbalssa) [inline src/components/Screens/ConfigurationScreen.tsx]:
```suggestion
        {translate('configuration_transaction_form')}
```

**Review by victorbalssa — COMMENTED** [review]
Thanks @hamza-fadil!

A few comment on translation and good to go 👍🏻

**Comment by hamza-fadil** [comment]
Hello, I applied the suggestions.
Thank you for your feedback.

**Review by victorbalssa — APPROVED** [review]

**Pull request merged**